packages/instant-meilisearch/src/adapter/search-response-adapter/pagination-adapter.ts

```diff
@@ -1,11 +1,11 @@
 import type {
-  MeiliSearchResponse,
+  MultiSearchResult,
   InstantSearchPagination,
   PaginationState,
 } from '../../types'
 
 function adaptNbPages(
-  searchResponse: MeiliSearchResponse<Record<string, any>>,
+  searchResponse: MultiSearchResult<Record<string, any>>,
```

Comment:
I find it weird reading `searchResponse: MultiSearchResult`. Are we receiving a response? A result? And for single or multi search?
Would it be worth it renaming `searchResponse` into `multiSearchResult`?
Feel free to ignore my comment if it's irrelevant :)

Reply:
Indeed! I was avoiding doing in this PR to avoid putting to many out of context changes. But it should definitely be changed